File: apps/docs/content/guides/auth/examples/managing-user-data/addingAndRetrievingUserData.dart
final res = await supabase.auth.signUp(
  email: 'valid.email@supabase.io',
  password: 'example-password',
  data: {
    'first_name': 'John',
    'age': 27,
  },
);

final User? user = supabase.auth.currentUser;
final Map<String, dynamic>? metadata = user?.userMetadata;

File: apps/docs/content/guides/auth/examples/managing-user-data/addingAndRetrievingUserData.swift
try await supabase.auth.signUp(
  email: "valid.email@supabase.io",
  password: "example-password",
  data: [
    "first_name": .string("John"),
    "age": .integer(27),
  ]
)

let user = try await supabase.auth.user()
let metadata = user.userMetadata